tests/unit/test_zowe_core.py

```diff
@@ -381,6 +386,38 @@ def test_profile_not_found_warning(self, get_pass_func):
             prof_manager.config_dir = self.custom_dir
             props: dict = prof_manager.load("non_existent_profile")
 
+    @patch("keyring.get_password", side_effect=keyring_get_password)
+    def test_profile_loading_with_env_variables(self, get_pass_func):
+        """
+        Test loading of correct file given a filename and directory,
+        also load by profile_name correctly populating fields from custom
+        profile and secure credentials
+        """
+        # Setup - copy profile to fake filesystem created by pyfakefs
+        custom_file_path = os.path.join(self.custom_dir, "zowe.config.json")
+        shutil.copy(self.original_nested_file_path, custom_file_path)
+        custom_file_path = os.path.join(self.custom_dir, "zowe.schema.json")
```

Comment:
@t1m0thyj I have added the test but the fake file system is not taking the `zowe.schema.json` file

Reply:
it would seem that the `pyfakefs` is not able to resolve relative paths.
We could make the paths absolute for the purpose of the tests, or find out if there is a way for it to load the schema file from `CWD` 😋